src/main/resources/reference.conf

```diff
@@ -143,4 +143,2022 @@ akka {
+              }
+              predecessor {
+                visible: true
+              }
+              timestamp {
+                visible: true
+              }
+              validation_pass {
+                visible: true
+              }
+              fitness {
+                visible: true
+              }
+              context {
+                visible: true
+              }
+              signature {
+                visible: true
+              }
+              protocol {
+                visible: true
+              }
+              chain_id {
+                visible: true
+              }
+              hash {
+                visible: true
+              }
+              operations_hash {
+                visible: true
+              }
+              period_kind {
+                visible: true
+              }
+              current_expected_quorum {
+                visible: true
+              }
+              active_proposal {
+                visible: true
+              }
+              baker {
+                visible: true
+              }
+              nonce_hash {
+                visible: true
+              }
+              consumed_gas {
+                visible: true
+              }
+              meta_level {
+                visible: true
+              }
+              meta_level_position {
+                visible: true
+              }
+              meta_cycle {
+                visible: true
+              }
+              meta_cycle_position {
+                visible: true
+              }
+              meta_voting_period {
+                visible: true
+              }
+              meta_voting_period_position {
+                visible: true
+              }
+              expected_commitment {
+                visible: true
+              }
+            }
+          }
+          fees {
+            visible: true,
+            attributes {
+              low {
+                visible: true
+              }
+              medium {
+                visible: true
+              }
+              high {
+                visible: true
+              }
+              timestamp {
+                visible: true
+              }
+              kind {
+                visible: true
+              }
+            }
+          }
+          operation_groups {
+            visible: true,
+            attributes {
+              protocol {
+                visible: true
+              }
+              chain_id {
+                visible: true
+              }
+              hash {
+                visible: true
+              }
+              branch {
+                visible: true
+              }
+              signature {
+                visible: true
+              }
+              block_id {
+                visible: true
+              }
+            }
+          }
+          operations {
+            visible: true,
+            attributes {
+              operation_id: {
+                visible: true
```

Comment:
visible should be false here

Reply:
For ``operation_id`` for all networks, I assume?